.circleci/config.yml

```diff
@@ -353,98 +361,94 @@ jobs:
     steps:
       - attach_workspace:
           at: artifacts
+      - run:
+          name: "Set tag in environment"
+          command: |
+            cat artifacts/bash.env >> "$BASH_ENV"
+            rm artifacts/bash.env
       - run:
           name: "Publish Release on GitHub"
-          environment:
-            GITHUB_TOKEN: $GITHUB_TOKEN
```

Comment:
There was also an issue with github authorization for publishing with `ghr`, removing this resolved it (adding `-t ${GITHUB_TOKEN}` to the `ghr` command was done in an earlier commit). :man_shrugging:

Reply:
Hmm, okay. I just didn't like the token being directly in the terminal which is why I removed the `-t ${GITHUB_TOKEN}`. Hopefully CircleCI never leaks it

Reply:
Yeah, this is how they do it in their tutorial too, so it *should* be okay. :crossed_fingers: